task_list.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
 
   gem.add_dependency "rack", "~> 1.0"
+  gem.add_dependency "nokogiri", "~> 1.6.0"
```

Comment:
I updated the gemspec to only pin the dependencies when the ruby version is too old.

```
ruby 1.9.3p551 (2014-11-13 revision 48407) [x86_64-linux]
Installing rack 1.6.5
Installing nokogiri 1.6.8.1
Installing activesupport 4.2.8
```

```
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
Using rack 2.0.1
Using nokogiri 1.7.1
Using activesupport 5.0.2
```